test/uk/gov/hmrc/cdsreimbursementclaimfrontend/journeys/RejectedGoodsScheduledJourneySpec.scala

```diff
@@ -33,11 +33,6 @@ import RejectedGoods.ValidationErrors._
 
 class RejectedGoodsScheduledJourneySpec extends AnyWordSpec with ScalaCheckPropertyChecks with Matchers {
 
-  implicit override val generatorDrivenConfig: PropertyCheckConfiguration =
```

Comment:
why this change?

Reply:
this is a valid ScalaCheck setting, why does it break things?

Reply:
It causes an exception to happen in the test runner